src/cmds.js

```diff
@@ -0,0 +1,185 @@
+
+// requires
+const config = require("../config.json");
+const logger = require("./logging.js").logger;
+
+
+/** the players bot commands */
+const commands = {
+
+    help: {
+        name: 'help',
+        usage: config.prefix + 'help [command]',
+        help: 'displays the command list with command summaries',
+
+        // args should be string[], message should be the discord Message as given in the message event
+        func: function (args, message) {
+
```

Comment:
As this file is likely to get very long, lets move Functionality out of this file and trigger the functionality from here. Keeping command functionality out of this file will keep it from getting absurdly long